.gitignore

```diff
@@ -3,6 +3,7 @@
 # compiled output
 /dist/
 /tmp/
+*.tgz
```

Comment:
@Turbo87 what are these `tgz` files?

Reply:
I looked through the `unpkg` link and didn't see them anywhere in the actual published package...

Reply:
the output of `npm pack`. I often run `npm pack` before `npm publish` to see what files are added to the published artifact.

Reply:
to avoid issues like https://github.com/mike-north/ember-api-actions/issues/410 😉

Reply:
Gotcha -- yeah, this configuration change looks right to me!